src/AccessibilityInsights.SharedUx/Controls/HierarchyControl.xaml.cs

```diff
@@ -852,34 +851,22 @@ public async void FileBug(HierarchyNodeViewModel vm = null)
                     { TelemetryProperty.IsAlreadyLoggedIn, BugReporter.IsConnected.ToString(CultureInfo.InvariantCulture) },
                 });
 
-                if (BugReporter.IsConnected && Configuration.SavedConnection?.IsPopulated == true)
+                if (BugReporter.IsConnected)
                 {
-                    Action<int> updateZoom = (int x) => Configuration.ZoomLevel = x;
-                    (int? bugId, string newBugId) = FileBugAction.FileNewBug(this.SelectedElement.GetBugInformation(BugType.NoFailure), Configuration.SavedConnection, Configuration.AlwaysOnTop, Configuration.ZoomLevel, updateZoom);
-
-                    vm.BugId = bugId;
-
-                    // Check whether bug was filed once dialog closed & process accordingly
-                    if (vm.BugId.HasValue)
+                    IssueInformation issueInformation = this.SelectedElement.GetIssueInformation(IssueType.NoFailure);
+                    FileBugAction.AttachIssueData(issueInformation, this.ElementContext.Id, this.SelectedElement.BoundingRectangle,
+                                this.SelectedElement.UniqueId);
+                    IIssueResult issueResult = FileBugAction.FileIssueAsync(issueInformation);
```

Comment:
do we need to `await` this async method?

Reply:
Discussed offline.